apps/etterlatte-saksbehandling-ui/client/src/components/behandling/handlinger/sendTilAttesteringModal.tsx

```diff
@@ -1,27 +1,28 @@
 import { BodyShort, Button, Heading, Modal } from '@navikt/ds-react'
 import { useState } from 'react'
 import { handlinger } from './typer'
-import { fattVedtak as fattVedtakApi } from '~shared/api/behandling'
-import { useNavigate, useParams } from 'react-router-dom'
+import { useNavigate } from 'react-router-dom'
 import { isFailure, isPending, useApiCall } from '~shared/hooks/useApiCall'
 import { ApiErrorAlert } from '~ErrorBoundary'
 import { FlexRow } from '~shared/styled'
+import { ApiResponse } from '~shared/api/apiClient'
 
-export const SendTilAttesteringModal = () => {
+export const SendTilAttesteringModal = ({
+  behandlingId,
+  fattVedtakApi,
+}: {
+  behandlingId: string
+  fattVedtakApi: (id: string) => Promise<ApiResponse<unknown>>
```

Comment:
Hva er tanken her? Kan ikke se at `fattVedtakApi` er brukt utenfor denne komponenten, så hvorfor ha den som prop?

Reply:
Fordi å fatte vedtak for tilbakekreving er et eget endepunkt så for å gjenbruke komponenten så med request injectes